README.md

```diff
@@ -1,6 +1,9 @@
 # insights-results-aggregator-utils
 Utilities for Insights Results Aggregator
 
+[![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/)
+[![forthebadge made-with-go](http://ForTheBadge.com/images/badges/made-with-go.svg)](https://go.dev/)
```

Comment:
I'd use [![forthebadge made-with-go](https://img.shields.io/badge/Made%20with-Go-1f425f.svg)](https://go.dev/) as the other badge is quite different and losses the aesthetic of the repo :smile:

Reply:
because Go is better :d

Reply:
updated